src/app/components/UserAssets/VestedAssets.tsx

```diff
@@ -99,60 +69,16 @@ export function VestedAssets() {
             )}
             {connected &&
```

Comment:
I would just extract this condition outside the render to some variable (e.g. `isVestingLoaded`) and use `useMemo`.